sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollationExpressionSuite.scala

```diff
@@ -167,7 +167,7 @@ class CollationExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
-  test("CollationKey generates correct collation key for collated string") {
+  ignore("CollationKey generates correct collation key for collated string") {
```

Comment:
why are we making this change in this PR?

Reply:
It is not the final 'answer' to this PR, I am still investigating. 😄